pagerduty/resource_pagerduty_event_orchestration_path_global_test.go

```diff
@@ -291,6 +296,27 @@ func testAccCheckPagerDutyEventOrchestrationGlobalDefaultConfig(t, ep, s, o stri
 	`)
 }
 
+func testAccCheckPagerDutyEventOrchestrationGlobalDefaultWithEscalationPolicy(t, ep, s, o string) string {
```

Comment:
This test should be renamed not to contain "Default" because a default global EO is an empty EO